src/components/base/translate/Translate.json

```diff
@@ -891,6 +891,28 @@
         "en": "Download file for EndNote, Zotero, Mendeley and ReferenceManager"
       }
     },
+    "multiReferences": {
+      "get-references": {
+        "da": "Hent referencer for %s materialer",
+        "en": "Get references for %s materials"
+      },
+      "choose-edition": {
+        "da": "Vælg udgave(r) for at hente referencer",
+        "en": "Choose edition(s) to get references"
+      },
+      "choose-edition-short": {
+        "da": "Vælg udgave",
+        "en": "Choose edition"
+      },
+      "missing-edition": {
+        "da": "Der er ikke valgt en specifik udgave for %s af materialerne",
+        "en": "You have not chosen a specific edition for %s of the materials"
+      },
+      "remove": {
+        "da": "Fjern",
```

Comment:
Tjek gerne om der er en translation ala denne allerede, vi vil gerne undgå at der skal oversættes det samme igen

Reply:
Jeg var i tvivl, jeg synes det er ikke så pænt at hente noget fra en helt anden kontext... Nu er de pushet til translations, men skal nok lade være med at tilføje ekstra translation næste gang.

Reply:
Ja. Der kan også være undtagelser. Hvis man fx ser et behov for teksten kunne være forskelligt alligevel. Det er en balance